gcloud/datastore/_implicit_environ.py

```diff
@@ -126,7 +122,8 @@ def get_connection():
     :rtype: :class:`gcloud.datastore.connection.Connection`
     :returns: A connection defined with the proper credentials.
     """
-    return get_scoped_connection(Connection, SCOPE)
+    credentials = get_credentials()
```

Comment:
We can defer if you like, but I had the impression that @jgeewax wanted to make it obvious everywhere how to use non-default credentials when getting a connection.